bestie/server/BUILD.bazel

```diff
@@ -49,3 +48,13 @@ container_push(
     # TODO: Change this tag to "live"
     tag = "testing",
 )
+
+cloud_run_deploy(
```

Comment:
ok for this PR: as mentioned, would be good to have a single step deployment, maybe using "multirun", or maybe without going through a container registry. Let me expand.

1) I don't think there's a good way to run "big query" locally. So, eg, for development and testing, developers will need to deploy all the time.

2) I would imagine a workflow where:
* Developer builds/deploys on cloud run all the time, his own instance, to see it working/testing, etc. Ideally, this does not require pushing a new container image all the time, if it does, it is some form of ephemeral table.
* When a new release is ready, a new container is prepared.
* Eventually it is deployed to staging and then production.

Reply:
> I don't think there's a good way to run "big query" locally

Maybe we can run locally, while talking to a shared bigquery instance?

Reply:
It's possible and maybe ok. I was doing this for astore, but there are two main drawbacks:

1) it's a bit annoying in that it requires creating a service account, *downloading* a credentials file that we absolutely cannot check in in the repository, plumbing in flags so we can pass it, and having some process so we can revoke it when team members leave. So eg, we'd need to provide instructions on how to get this file, and be diligent in managing it.

2) when running locally .. it's a different environment than on cloud run/GAE. Can be small things like logging/monitoring not being available/searchable, to things like gRPC behaving slightly differently because there is no proxy in front of it. It can be desireable at times, but can also be a source of bugs/issues.

Reply:
Thinking about this more:
* The only way I can find to deploy to Cloud Run via bazel is via the `gcloud run deploy` command
* `gcloud run deploy` can deploy given an image and version (used here) or the `--source` parameter, which tries to build from the CWD by looking for a Dockerfile or using buildpacks. Bazel integration doesn't look promising here.
* As a result, we will always have to push before deploying
* Therefore, we probably need to parameterize both the service name and image tag, so there are no collisions between iteration cycles. Putting these in the BUILD file is suboptimal, as if someone forgets to change it before running they will clobber something else. We can parameterize the new deploy rule with a custom flag, requiring the dev to specify it on the commandline, but we can't with the container push rule, since we don't control it (we'd have to fork)

So I'm thinking (if this is necessary):
* leave the image tag at "latest" - latest will always get clobbered but hopefully that doesn't happen often
* add a flag for the developer to set the service suffix (required) - so a prod push would require e.g. `bazel run //bestie/server:bestie_cloud_run_deploy --//f/cloud_run:service_suffix=prod` - allows for arbitrary "environments"
* use multirun to reduce the possibility of races

...or if testing locally against a remote bigquery instance is possible, maybe we can punt on the flag, and just have a "staging" and "prod" rule instances. I think this will become more clear after initial bigquery support is added, so let's punt until then?

Reply:
sgtm